vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/PackageId.java

```diff
@@ -209,14 +209,16 @@ public PackageId(String group, String name, String version) {
     public PackageId(String group, String name, Version version) {
         fromPath = false;
         // validate group
-        if (group.equals(ETC_PACKAGES)) {
+        if (group == null || group.equals(ETC_PACKAGES)) {
```

Comment:
this change is not related to the issue.

Reply:
removing, will make separate ticket for this